### DISCUSSION TITLE
.Save() and increment broken on big tables?

### DISCUSSION THREAD

**Discussion by githubUser982** [sequelize/sequelize#15219] [Help & Questions] [answered]
I have a big table (1.3m rows.)
It's been working fine and dandy until recently where .save() does not do anything (No error.) but ONLY on the newer rows.

I've tested the latest version, 6.25.3, same problem.

increment is broken as well (Doesn't save)
item.changed() displays correctly
Only table.update() with where clause works for these entries.

**Comment by ephys** [comment]
What's the DataType of your primary key?

**Comment by ephys** [comment, marked as answer]
https://github.com/sequelize/sequelize/issues/15220#event-7765467182
